pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "pipelinedp"
+version = "0.1.0"
+description = ""
+authors = ["Chinmay Shah <chinmayshah3899@gmail.com>", "Vadym Doroshenko <dvadym@google.com>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+apache-beam = "^2.29.0"
+pyspark = "^3.1.1"
+absl-py = "^0.12.0"
+dataclasses = {version = "^0.8", python = "~3.6"}
+numpy = "<1.21"
```

Comment:
Just wondering, why <1.21 version limitation? (which currently cover all current versions). Maybe it's better to specify lower bound (something like more than 1.17 or 1.20).

Reply:
Numpy has dropped support for python 3,6 starting 1.21.

Reply:
Thanks , it makes sense.